doc/source/dev/roadmap-detailed.rst

```diff
@@ -71,49 +59,67 @@ add new benchmarks, however running the benchmarks is not very intuitive.
 Making this easier is a priority.
 
 
+Moving to the Meson build system
+````````````````````````````````
+Support for the Meson build system was merged into SciPy master in Dec 2021.
+This significantly improves build performance, and will fix multiple issues
+(e.g., our issues with Windows compilers, cross-compilation support). The aim
+is to make it the default build system for SciPy 1.9.0, and then remove support
+for ``numpy.distutils``/``setuptools`` in SciPy 1.10.0. For more details, see
+`gh-13615 <https://github.com/scipy/scipy/issues/13615>`_.
+
+
 Use of Cython
 `````````````
-Regarding Cython code:
-
-- It's not clear how much functionality can be Cythonized without making the
-  .so files too large.  This needs measuring.
-- Cython's old syntax for using NumPy arrays should be removed and replaced
-  with Cython memoryviews.
+Cython's old syntax for using NumPy arrays should be removed and replaced with
+Cython memoryviews. When Cython 3.0 is released, the last use of the deprecated
```

Comment:
I wonder if we will genuinely be able to replace the old syntax completely without suffering performance hits anytime soon. See i.e., discussion in: https://github.com/scipy/scipy/issues/12379

Maybe this is improving, though for pure array pass-through situations `ndarray` seemed to be outperforming memoryviews a bit.

Reply:
Yeah, I remember. This was already there though, and makes sense in general (if not high-prio). So I left that bit as is.